integration-tests/csimple/pom.xml

```diff
@@ -101,7 +101,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.camel.quarkus</groupId>
-            <artifactId>camel-quarkus-xml-io-dsl-deployment</artifactId>
```

Comment:
I just point that in case it is a leftover.

Reply:
yes, it belongs to some previous commit. I'll leave it here and will silently shame in the corner.